src/aposteriori/data_prep/create_frame_data_set.py

```diff
@@ -1266,13 +1329,22 @@ def make_frame_dataset(
         into a 3x3x3 gaussian density using the formula indicated by Zhang et al., (2019) ProdCoNN.
 
         https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fprot.25868&file=prot25868-sup-0001-AppendixS1.pdf
-
+    blacklist_csv: StrOrPath
+        Path to blacklist csv file.
 
     Returns
     -------
     output_file_path: pathlib.Path
         A path to the location of the output dataset.
     """
+    # Filter by blacklist:
+    if blacklist_csv and pathlib.Path(blacklist_csv).exists():
+        structure_files = filter_structures_by_blacklist(
+            structure_files, pathlib.Path(blacklist_csv)
+        )
+    elif blacklist_csv and pathlib.Path(blacklist_csv).exists() == False:
```

Comment:
I'm not sure if this line will be evaluated properly I'd change it to `elif blacklist_csv and (pathlib.Path(blacklist_csv).exists() == False):` to make sure it does.

Reply:
I made a suggestion below this, so it's no longer as relevant!